packages/suite-native-settings/src/screens/SettingsAdvancedScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import { Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const screenStyle = prepareNativeStyle(() => ({
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+}));
+
+export const SettingsAdvancedScreen = () => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <View style={[applyStyle(screenStyle)]}>
```

Comment:
Please create some component in navigation package called `Screen` or smthing like that and wrap all screens to that component